cypress/integration/features/dashboard/dashboard.feature

```diff
@@ -1,14 +1,82 @@
+    And I see time input field
+    And I click on Save button
+    Then I see news is created successfully with title 'this is a team news' and with desccription 'test team news description'
+
+  Scenario: as a pre-condition teacher adds student as team member
+    Given I am logged in as a 'teacher' at 'brb'
+    When I go to teams overview
+    And I go to a team 'Musik'
+    And I click on three dot menu on the team title
+    And I click on manage team members option
+    And I click on Add internal attendees button
+    And new dialog opens to select student 'Herbert Kraft' from the drop down list
+    And I click on add button
+   Then I see the student named 'Herbert Kraft' on the team members table
+
+  Scenario: student arrives on dashboard
     Given I am logged in as a 'student' at 'brb'
     When I arrive on the dashboard
-    Then I can see the welcome message
+    Then I see the welcome message 'Hallo Herbert Kraft!'
+    Then I see school news with title 'this is a school news' and description 'test school news description'
+    Then I see teams news with title 'this is a team news' and description 'test team news description'
+    Then I can see the assigned task 'Task11'
+
+  Scenario: teacher arrives on dashboard
+    Given I am logged in as a 'teacher' at 'brb'
+    When I arrive on the dashboard
+    Then I see the welcome message 'Hallo Karl Herzog!'
+    Then I see school news with title 'this is a school news' and description 'test school news description'
+    Then I see teams news with title 'this is a team news' and description 'test team news description'
+    Then I can see the assigned task 'Task11'
+    Then I can see the draft task 'Task1'
 
-  Scenario: Check tasks on the dashboard
+  Scenario: as a post-condition teacher deletes the school news
     Given I am logged in as a 'teacher' at 'brb'
     When I arrive on the dashboard
-    Then I can see the assigned tasks
-    Then I can see the draft tasks
\ No newline at end of file
+    And I click on the news teaser 'this is a school news'
+    When I click on delete button
+    And I confirm the deletion on confirmation dialog box
+    Then I do not see the news 'this is a school news'
+
+  Scenario: as a post-condition teacher deletes the team news
+    Given I am logged in as a 'teacher' at 'brb'
+    When I arrive on the dashboard
+    And I click on the news teaser 'this is a team news'
+    When I click on delete button
+    And I confirm the deletion on confirmation dialog box
+    Then I do not see the news 'this is a team news'
+
+  Scenario: as a post-condition teacher deletes the student as a  team member
+    Given I am logged in as a 'teacher' at 'brb'
+    When I go to teams overview
+    And I go to a team 'Musik'
+    And I click on three dot menu on the team title
+    And I click on manage team members option
+    When I select the student 'Herbert Kraft' and click on delete icon
+    Then I see 'Herbert Kraft' is not visible on the table
```

Comment:
Sugguestion: Scenario "Student does not see news anymore on dashboard" could be added.